packages/xo-server/src/xo-mixins/metadata-backups.js

```diff
@@ -874,29 +874,20 @@ export default class metadataBackup {
   async deleteMetadataBackup(id: string) {
     const app = this._app
     const [remoteId, ...path] = id.split('/')
-    const metadataFolder = path.join('/')
-    const { proxy, url, options } = await app.getRemoteWithCredentials(remoteId)
-    if (proxy !== undefined) {
-      await app.callProxyMethod(proxy, 'backup.deleteMetadataBackup', {
-        backupId: metadataFolder,
-        remote: { url, options },
+    const backupId = path.join('/')
+    const remote = await app.getRemoteWithCredentials(remoteId)
+
+    let backupType
+    if (remote.proxy !== undefined) {
+      backupType = await app.callProxyMethod(remote.proxy, 'backup.deleteMetadataBackup', {
```

Comment:
Unfortunately I don't think it will work for the proxy.

At least it does not work with older versions of it.

Maybe `@xen-orchestra/backups` should provide a helper to guess the backup type?